### PR TITLE
Chore/code cleanup

### PR DESCRIPTION
**Pull request by maxjerin**
Add prettier script and husky pre-commit hook.
Remove hardcoded username and password and load them from dotenv instead.
Run prettier on all files.
Add Readme for starting app locally.

### CODE REVIEW

File: src/components/AuthForm.js [hunk 1]
```diff
@@ -22,8 +22,8 @@ class AuthForm extends CommunisComponent {
     this.handleSubmit = this.handleSubmit.bind(this);
     this.state = {
       form: {
-        username: null,
-        password: null,
+        username: process.env.REACT_APP_DEV_USER,
+        password: process.env.REACT_APP_DEV_PASSWORD,
```
Comment (jmathewt) [inline src/components/AuthForm.js:26]: Nice! These will be environment variables

File: src/pages/AuthPage.js [hunk 2]
```diff
@@ -1,81 +1,83 @@
...
-        this.props.history.push('/');
-    };
+  handleLoginSubmit = authentication => {
+    this.props.dispatch(
+      authenticateUser(authentication.username, authentication.password),
+    );
+  };
 
-    handleLoginSubmit = (authentication) => {
-        this.props.dispatch(authenticateUser("admin@communis.com", "A@12word"));
```
Comment (maxjerin) [inline src/pages/AuthPage.js:24]: @jobinthomasnypl, the main idea of .env was to remove this.

File: src/components/AuthForm.js (hunk 1 above)
Comment (maxjerin) [reply src/components/AuthForm.js:26]: You know you can try and find a new username.https://help.github.com/en/enterprise/2.13/user/articles/changing-your-github-username

File: src/pages/AuthPage.js (hunk 2 above)
Comment (robinvk6) [reply src/pages/AuthPage.js:24]: @maxjerin Can you please resolve the conflicts?

**Pull request merged**